ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Swap_File/tasks/main.yml

```
@@ -9,6 +9,25 @@
 # Not installed: When swap_root.rc or swap_tmp.rc does NOT equal 0
 # Assume swap path of /swapfile override where applicable
 
+# Set Filesystem Check Fact, In Case No Root Filesystem
+- set_fact: btrfs_fs_found="false"
```

linter has failed on this - let's add a name to this:
```suggestion
- name: Intialise btrfs variable
  set_fact: btrfs_fs_found="false"
```